### PR TITLE
Update introduction.mdx

### PR DESCRIPTION
**Pull request by Nestpebble**
A few changes to increase clarity

### CODE REVIEW

File: docs/about/introduction.mdx [hunk 1]
```diff
@@ -12,7 +12,8 @@ Meshtastic® is a project that enables you to use inexpensive LoRa radios as a l
 ### Features
 
 - Long range ([_166km record by PuzzledPancake_](/docs/overview/range-tests#current-record))
-- No phone required for mesh communication
+- No phone signal required for mesh communication
```
Comment (sachaw) [inline docs/about/introduction.mdx:15]: I'd Rever this, as the functionality of the mess does not depend on a phone at all

File: docs/about/introduction.mdx [hunk 2]
```diff
@@ -21,11 +22,11 @@ Meshtastic® is a project that enables you to use inexpensive LoRa radios as a l
 
 ## How it works
 
-The underlying technology, LoRa, is a long range radio protocol available to most regions without requiring certification like HAM operators.
+The underlying technology, LoRa, is a long range radio protocol available to most regions without requiring additional licensing or certification, like HAM operators.
 
-The radios automatically rebroadcast messages they receive in order to create a mesh network so that everyone in the group can receive messages - even from the furthest member. Depending on settings used, the Meshtastic mesh can sustain up to 80 device nodes.
+The radios automatically link up and rebroadcast messages they receive in order to create a mesh network so that everyone in the group can receive messages - even from the furthest member. Depending on settings used, the Meshtastic mesh can sustain up to 80 device nodes.
```
Comment (sachaw) [inline docs/about/introduction.mdx:27]: Don't like the term "link upp" I think we can leave this part as-is

File: docs/about/introduction.mdx [hunk 3]
```diff
@@ -21,11 +22,11 @@ Meshtastic® is a project that enables you to use inexpensive LoRa radios as a l
...
 
-Radios can be paired to a single phone so that your friends and family are able to address a message to your specific radio. Each device supports a connection from a single user at a time.
+Meshtastic radios are paired to a single phone so that your friends and family are able to address a message to your specific radio. Each device supports a connection from a single user at a time.
```
Comment (sachaw) [inline docs/about/introduction.mdx:29]: I think this should be referred, as phones are only one option to interact with the mesh

**Review by sachaw — CHANGES_REQUESTED** [review]

File: docs/about/introduction.mdx (hunk 1 above)
Comment (Nestpebble) [reply docs/about/introduction.mdx]: accepted

File: docs/about/introduction.mdx (hunk 2 above)
Comment (Nestpebble) [reply docs/about/introduction.mdx]: accepted

File: docs/about/introduction.mdx (hunk 3 above)
Comment (Nestpebble) [reply docs/about/introduction.mdx]: accepted

**Review by Nestpebble — COMMENTED** [review]
accepted the changes - pull request updated (I think...)

**Review by caveman99 — APPROVED** [review]

**Review by sachaw — APPROVED** [review]

**Pull request merged**